index.html

```diff
@@ -288,34 +288,16 @@
     </p>
 
     <h2>Installing</h2>
-    <p>
-    Assuming you have the prerequisites and are running <code>bash</code>:
-    </p>
-    <pre class="code">
-curl -O https://sw.lsstcorp.org/eupspkg/newinstall.sh
-bash newinstall.sh
-
-source loadLSST.bash
-
-eups distrib install -t v12_0 lsst_apps</pre>
-    <p>
-    This will download and build from source a specific release of the LSST
-    Stack (v12.0, in the example above).
-    </p>
 
-    <p>
-    For complete instructions, see the <a href="https://pipelines.lsst.io/">documentation</a>.
-    </p>
+    <p>There are several ways of installing the LSST Stack, including from source, through Anaconda, or as Docker containers. Our <a href="https://pipelines.lsst.io/install/index.html">installation documentation</a> will get you started.</p>
```

Comment:
I think the intent of the original text was to provide one way, with as few commands as possible, to demonstrate that it's not terribly difficult and as an encouragement to try it.  Pointing to another page doesn't really accomplish those goals.